src/Authorization.php

```diff
@@ -40,12 +40,12 @@ public function __construct(HubRegistry $registry, ?int $cookieLifetime = null)
     /**
      * Sets mercureAuthorization cookie for the given hub.
      *
-     * @param string[]    $subscribe        a list of topics that the authorization cookie will allow subscribing to
-     * @param string[]    $publish          a list of topics that the authorization cookie will allow publishing to
-     * @param mixed[]     $additionalClaims an array of additional claims for the JWT
-     * @param string|null $hub              the hub to generate the cookie for
+     * @param string[]|string      $subscribe        a list of topics that the authorization cookie will allow subscribing to
```

Comment:
```suggestion
     * @param string[]|string      $subscribe        a topic or a list of topics that the authorization cookie will allow subscribing to
```
to make it clear what `|string` means, same for others if you agree